<UI version="4.0" >
 <class>MyWidgetClass</class>
 <widget class="QWidget" name="MyWidgetClass" >
	 <property name="geometry">
		 <rect>
			 <x>0</x>
			 <y>0</y>
			 <width>800</width>
			 <height>516</height>
		 </rect>
	 </property>
	 <property name="windowTitle">
		 <string>MyWidget</string>
	 </property>
	 <widget class="QLineEdit" name="editSearch">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>130</y>
				 <width>381</width>
				 <height>31</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QLabel" name="label">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>100</y>
				 <width>231</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>12</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>S1: Основная строка</string>
		 </property>
		 <property name="buddy">
			 <cstring>editSearch</cstring>
		 </property>
	 </widget>
	 <widget class="QLineEdit" name="editSubject">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>220</y>
				 <width>381</width>
				 <height>31</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QLabel" name="label_2">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>190</y>
				 <width>231</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>12</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>S2: Строка для поиска</string>
		 </property>
		 <property name="buddy">
			 <cstring>editSubject</cstring>
		 </property>
	 </widget>
	 <widget class="QLineEdit" name="editReplace">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>300</y>
				 <width>381</width>
				 <height>31</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QLabel" name="label_3">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>270</y>
				 <width>231</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>12</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>S3: Строка для замены</string>
		 </property>
		 <property name="buddy">
			 <cstring>editReplace</cstring>
		 </property>
	 </widget>
	 <widget class="QLabel" name="lblOutput">
		 <property name="geometry">
			 <rect>
				 <x>50</x>
				 <y>370</y>
				 <width>691</width>
				 <height>111</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <family>Lucida Console</family>
				 <pointsize>13</pointsize>
				 <weight>50</weight>
				 <italic>false</italic>
				 <bold>false</bold>
				 <underline>false</underline>
				 <strikeout>false</strikeout>
				 <kerning>true</kerning>
			 </font>
		 </property>
		 <property name="styleSheet">
			 <string notr="true">background-color: black; color: white;</string>
		 </property>
		 <property name="frameShape">
			 <enum>QFrame::Box</enum>
		 </property>
		 <property name="frameShadow">
			 <enum>QFrame::Raised</enum>
		 </property>
		 <property name="text">
			 <string>output...</string>
		 </property>
		 <property name="alignment">
			 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
		 </property>
		 <property name="wordWrap">
			 <bool>true</bool>
		 </property>
	 </widget>
	 <widget class="QGroupBox" name="groupBoxOccurs">
		 <property name="geometry">
			 <rect>
				 <x>470</x>
				 <y>110</y>
				 <width>271</width>
				 <height>161</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>11</pointsize>
			 </font>
		 </property>
		 <property name="title">
			 <string>Заменить вхождения S2 в S1 на S3:</string>
		 </property>
		 <widget class="QRadioButton" name="radioFirstOccur">
			 <property name="geometry">
				 <rect>
					 <x>20</x>
					 <y>40</y>
					 <width>241</width>
					 <height>17</height>
				 </rect>
			 </property>
			 <property name="text">
				 <string>Первое вхождение</string>
			 </property>
		 </widget>
		 <widget class="QRadioButton" name="radioLastOccur">
			 <property name="geometry">
				 <rect>
					 <x>20</x>
					 <y>80</y>
					 <width>241</width>
					 <height>17</height>
				 </rect>
			 </property>
			 <property name="text">
				 <string>Последнеее вхождение</string>
			 </property>
		 </widget>
		 <widget class="QRadioButton" name="radioAllOccur">
			 <property name="geometry">
				 <rect>
					 <x>20</x>
					 <y>120</y>
					 <width>241</width>
					 <height>17</height>
				 </rect>
			 </property>
			 <property name="text">
				 <string>Все вхождения</string>
			 </property>
		 </widget>
	 </widget>
	 <widget class="QPushButton" name="btnStart">
		 <property name="geometry">
			 <rect>
				 <x>470</x>
				 <y>290</y>
				 <width>271</width>
				 <height>41</height>
			 </rect>
		 </property>
		 <property name="text">
			 <string>Применить</string>
		 </property>
	 </widget>
 </widget>
 <layoutDefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources>
   <include location="mywidget.qrc"/>
 </resources>
 <connections/>
</UI>
